rhizome/postgres/bin/metrics-collector

```diff
@@ -22,16 +22,19 @@ fail "Metrics config file does not exist: #{metrics_dir}/config.json" unless Fil
 FileUtils.mkdir_p("#{metrics_dir}/pending")
 FileUtils.mkdir_p("#{metrics_dir}/done")
 
+# Clear the pending directory on startup (anything there is from a crash)
+Dir.children("#{metrics_dir}/pending").each { File.delete("#{metrics_dir}/pending/#{_1}") }
+
 # Parse config
 metrics_config_hash = JSON.parse(File.read("#{metrics_dir}/config.json"))
 endpoints = metrics_config_hash["endpoints"]
-max_pending_buffer_size = metrics_config_hash["max_pending_buffer_size"] || 100
+max_file_retention = metrics_config_hash["max_file_retention"] || 100
```

Comment:
Found this while testing: there's another reference in `model/postgres/posrgres_server.rb` to this config

```diff
diff --git a/model/postgres/postgres_server.rb b/model/postgres/postgres_server.rb
index 05360c11..a04660dc 100644
--- a/model/postgres/postgres_server.rb
+++ b/model/postgres/postgres_server.rb
@@ -253,7 +253,7 @@ class PostgresServer < Sequel::Model
       endpoints: [
         "https://localhost:9090/federate?match[]=%7B__name__%3D~%22.%2B%22%7D"
       ],
-      max_pending_buffer_size: 120,
+      max_file_retention: 120,
       interval: "15s",
       additional_labels: {},
       metrics_dir: "/home/ubi/postgres/metrics"
```

Reply:
I fixed this and tested this, works fine